recipes/install_ext_services.rb

```diff
@@ -22,8 +22,8 @@
 
 case node['abiquo']['profile']
 when "monolithic", "server"
-    packages = %w{MariaDB-server MariaDB-client redis rabbitmq-server}
-    services = %w{mysql redis rabbitmq-server}
+    packages = %w{MariaDB-server MariaDB-client redis rabbitmq-server cronie}
```

Comment:
Verify that the additional package is installed in [this test](https://github.com/abiquo/abiquo-cookbook/blob/master/spec/install_ext_services_spec.rb#L29-L46).